### PR TITLE
#49 Added Google Auth

### LINKED ISSUE #49: add google login support

### LINKED ISSUE #117: dont expose social accesstokens when using include=xxx api

### PR DESCRIPTION
**Pull request by Dhroov7**
fixes #49 
fixes #117

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
     "passport-custom": "^1.0.5",
     "passport-facebook": "^2.1.1",
     "passport-github2": "^0.1.10",
+    "passport-google-oauth": "^1.0.0",
```
Comment (championswimmer) [inline package.json]: I think `passport-google-oauth20` is the most popular one

File: secrets-sample.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
   "TWITTER_CONSUMER_SECRET": "dG5gG5Sk5ZgMlHO7D9p9A6jwphmcB8NKsf0witZQb2FqIsjaIE",
   "GITHUB_CONSUMER_KEY": "Iv1.1a76b0413117bc79",
   "GITHUB_CONSUMER_SECRET": "8b0ca842b105493ab9c14bb3089cda5ddb603e56",
+  "GOOGLE_CLIENT_ID":"1012935287725-dpsdd66dkgr3m0c5fpp8h41jn6b7jn5q.apps.googleusercontent.com",
```
Comment (championswimmer) [inline secrets-sample.json]: don't expose your own ones here. just put some random strings here

Comment (Dhroov7) [reply secrets-sample.json]: @championswimmer Done!

File: src/server.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 /**
  * Created by championswimmer on 08/03/17.
  */
-require('newrelic')
```
Comment (championswimmer) [inline src/server.js]: don't remove the require('newrelic') line

Comment (championswimmer) [reply src/server.js]: It is required for development, but on production, newrelic is needed

**Comment by championswimmer** [comment]
You need to remove the last commit. 

Best way is 

```
git checkout master
git checkout 36e6f50
git branch -f master
git checkout master
git push origin master -f
```

**Comment by Dhroov7** [comment]
@championswimmer Done!

File: src/routers/connect/index.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ const router = require('express').Router()
 router.use('/', require('./local'))
 router.use('/facebook', require('./facebook'))
 router.use('/twitter', require('./twitter'))
+router.use('/google', require('./google'))
```
Comment (championswimmer) [inline src/routers/connect/index.js]: @Dhroov7 you had missed this

**Pull request merged**